proto/internalapi/sensor/collector.proto

```diff
@@ -37,3 +39,46 @@ message CollectorConfig {
 
   Networking networking = 1;
 }
+
+message ProcessSignal {
+  // A unique UUID for identifying the message
+  // We have this here instead of at the top level
+  // because we want to have each message to be
+  // self contained.
+  string id = 1;
+
+  // ID of container associated with this process
+  string container_id = 2;
+
+  // Process creation time
+  google.protobuf.Timestamp time = 3;
```

Comment:
I was going through the timestamps for network flows recently and I needed help of human to understand which one means what.
To avoid similar issues, maybe we could name this one `creation_time` instead of only `time`? That would make it specific and thus harder to misunderstand.

Reply:
Done.